docker/run_container_release.sh

```diff
@@ -14,6 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -x
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```

Comment:
This is duplicated in a few places. Not a deal breaker, but probably worth consolidating.

Reply:
Cant really be consolidated since its local to the file being run. If you put it in a different file, the value would change.